config/jest/environment/jsdom.mjs

```diff
@@ -0,0 +1,19 @@
+import { TestEnvironment } from 'jest-environment-jsdom'
+
+/**
+ * Virtual browser environment
+ * Adds jsdom window/document globals
+ */
+class BrowserVirtualEnvironment extends TestEnvironment {
+  async setup () {
+    await super.setup()
+
+    // Access virtual console
+    const { virtualConsole: console } = this.dom
```

Comment:
**nitpick:** Given the rename I think it'd be easier to read if we just used a normal assignment here, unless there's a particular reason to destructure that I'm missing:

```suggestion
    // Access virtual console
    const console = this.dom.virtualConsole
```

Reply:
I'm happy to not rename?

```mjs
const { virtualConsole } = this.dom
```

Reply:
That works too 👍🏻

Reply:
Done